app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
```

Comment:
indentation wrong
")" on another line

Reply:
@skmiwa1  if self on next line - yes, but not in this situation

Reply:
name on the next line! line 7